static/docs/command-reference/remote/add.md

```diff
@@ -23,20 +23,20 @@ positional arguments:
 
 ## Description
 
-`name` and `url` are required. `url` specifies a location to store your data. It
-can be an SSH, S3 path, Azure, Google Drive path, Google Cloud path, Aliyun OSS,
-local directory, etc. (See all the supported remote storage types in the
-examples below.) If `url` is a local relative path, it will be resolved relative
-to the current working directory but saved **relative to the config file
-location** (see LOCAL example below). Whenever possible DVC will create a remote
-directory if it doesn't exists yet. It won't create an S3 bucket though and will
-rely on default access settings.
-
-> If you installed DVC via `pip`, depending on the remote storage type you plan
-> to use you might need to install optional dependencies: `[s3]`, `[ssh]`,
-> `[gs]`, `[azure]`, `[gdrive]`, and `[oss]`; or `[all]` to include them all.
-> The command should look like this: `pip install "dvc[s3]"`. This installs
-> `boto3` library along with DVC to support Amazon S3 storage.
+`name` and `url` are required. `url` specifies a location (path, address,
```

Comment:
not sure about this part with `path, address, endpoint` .. i think it was simple and good enough - location